doc/nrf/app_memory.rst

```diff
@@ -84,6 +84,37 @@ Besides applying `General recommendations`_, you can also complete the following
 * Reduce the verbosity of assert messages by setting :kconfig:option:`CONFIG_ASSERT_VERBOSE` to ``n``.
 * Check `Thread`_ memory footprint optimization actions, as the Matter application layer uses OpenThread.
 
+.. _app_memory_nfc:
+
+NFC
+***
+
+To optimize your application that supports the NFC tag functionality follow the `General recommendations`_.
+The NFC :ref:`type_2_tag` and :ref:`type_4_tag` libraries do not provide configuration options that affect memory usage in an application, so the only ways to optimize memory usage relevant to NFC are the following:
+
+* Reduce logging level in the NFC integration module by setting :kconfig:option:`CONFIG_NFC_PLATFORM_LOG_LEVEL_CHOICE`
+
+Additionally, if your applicatin uses the :ref:`type_4_tag` library:
+
+  * Set the maximum NDEF file size using :kconfig:option:`CONFIG_NDEF_FILE_SIZE` if the read-write mode is supported,
+  * Set the maximum size of the buffer for NDEF message exchange using :kconfig:option:`CONFIG_NFC_TNEP_RX_MAX_RECORD_CNT` and :kconfig:option:`CONFIG_NFC_TNEP_RX_MAX_RECORD_SIZE` if the TNEP protocol is supported.
+
+To optimize your application that supports the NFC poller functionality using the :ref:`st25r3911b_nfc_readme` perform the following steps:
+
+* Reduce logging level of the NFC components used in the application, for example: :kconfig:option:`CONFIG_ST25R3911B_LIB_LOG_LEVEL_CHOICE`, :kconfig:option:`CONFIG_NFC_T4T_ISODEP_LOG_LEVEL_CHOICE`,
+* Set buffer sizes in the NFC T4T protocol implementation for the poller side: :kconfig:option:`CONFIG_NFC_T4T_HL_PROCEDURE_CC_BUFFER_SIZE`, :kconfig:option:`CONFIG_NFC_T4T_HL_PROCEDURE_APDU_BUF_SIZE`,
```

Comment:
```suggestion
* Reduce the logging level of the NFC components used in the application, for example, using the options :kconfig:option:`CONFIG_ST25R3911B_LIB_LOG_LEVEL_CHOICE` and :kconfig:option:`CONFIG_NFC_T4T_ISODEP_LOG_LEVEL_CHOICE`.
* Set the buffer sizes in the NFC T4T protocol implementation for the poller side using the options :kconfig:option:`CONFIG_NFC_T4T_HL_PROCEDURE_CC_BUFFER_SIZE` and :kconfig:option:`CONFIG_NFC_T4T_HL_PROCEDURE_APDU_BUF_SIZE`.
```

Reply:
Fixed, but I just left the `CONFIG_ST25R3911B_LIB_LOG_LEVEL_CHOICE` option because there are a few of them, and didn't want this part to be too long. Should I list all options here?